mergify_engine/worker.py

```diff
@@ -907,6 +907,24 @@ def get_process_index_from_env() -> int:
         return 0
 
 
+async def ping_redis(
+    redis: typing.Union[redis_utils.RedisStream, redis_utils.RedisCache],
+    redis_name: str,
+) -> None:
+    def retry_log(retry_state: tenacity.RetryCallState) -> None:
+        statsd.increment("redis.client.connection.errors")
+        LOG.warning(
+            f"Couldn't connect to Redis {redis_name}, retrying...",
+        )
+
+    r = tenacity.retry(
+        wait=tenacity.wait_exponential(multiplier=0.2),
```

Comment:
We need to set a max= otherwise the exponential can become too big, like max 5 seconds